articles/tokens/concepts/refresh-tokens.md

```diff
@@ -26,41 +26,29 @@ The Refresh Token behavior is applicable to [OIDC-conformant applications](/api-
-
-* A Single-Page Application (normally implementing [Implicit Flow](/flows/concepts/implicit)) should not ever receive a Refresh Token. A Refresh Token is essentially a user credential that allows a user to remain authenticated indefinitely. This sensitive information should be stored securely and *not* exposed client-side in a browser.
-
-* If you are implementing an SPA using [Implicit Flow](/flows/concepts/implicit) and you need to renew a token, the only secure option for doing so is to use [Silent Authentication](/api-auth/tutorials/silent-authentication).
-
-* If you limit offline access to your API, a safeguard configured via the **Allow Offline Access** switch on the [API Settings](${manage_url}/#/apis), Auth0 will not return a Refresh Token for the API (even if you include the `offline_access` <dfn data-key="scope">scope</dfn> in your request).
-
-### Rules
-
-Rules will run for the [Refresh Token Exchange](/tokens/guides/use-refresh-tokens). To execute special logic, you can look at the `context.protocol` property in your rule. If the value is `oauth2-refresh-token`, then the rule is running during the exchange.
-
-::: warning
-If you try to do a <a href="/rules/redirect">redirect</a> with <code>context.redirect</code>, the authentication flow will return an error.
-:::
-
-### Custom claims
-
-If you have added custom claims to your tokens using a rule, the custom claims will appear in new tokens issued when using a Refresh Token for as long as your rule is in place. Although new tokens do not automatically inherit custom claims, rules run during the Refresh Token flow, so the same code will be executed. This allows you to add or change custom claims in newly-issued tokens without forcing previously-authorized applications to obtain a new Refresh Token.
-
 ## SDK support
 
 ### Web apps
 
-All our main SDKs support Refresh Tokens out of the box. Some are [Node.js](/quickstart/webapp/nodejs), [ASP.NET Core](/quickstart/webapp/aspnet-core), [PHP](/quickstart/webapp/php), [Java](/dev-centers/java), and many more. For a complete listing, see [Quickstarts](/quickstart/webapp).
+Auth0 SDKs support Refresh Tokens out of the box including: 
+
+- [Node.js](/quickstart/webapp/nodejs)
+- [ASP.NET Core](/quickstart/webapp/aspnet-core)
+- [PHP](/quickstart/webapp/php)
+- [Java](/dev-centers/java)
+
+For a complete listing, see [Quickstarts](/quickstart/webapp).
 
 ### Single-page apps
 
 [Silent Authentication](/api-auth/tutorials/silent-authentication) is the method that is used to refresh a token for web apps that execute in a browser. [Auth0.js](/libraries/auth0js), our client-side library, provides methods for this functionality:
```

Comment:
> Auth0.js, our client-side library

I know this wasn't something that was changed, but should we alter the wording here to posit that this SDK is older and has effectively been superseded? I don't think these are the right words, but something to the effect of "Auth0.js, our older JS SDK.."